src/angular/src/app/spark-core-angular/components/sprk-ordered-list/sprk-ordered-list.component.spec.ts

```diff
@@ -28,6 +28,18 @@ describe('SparkOrderedListComponent', () => {
     expect(element.classList.toString()).toEqual(component.getClasses());
   });
 
+  it('should add the correct classes if bare is set as listType', () => {
```

Comment:
Did we miss some list tests or is this related to buttons?

Reply:
missed a few lines to test